docs/gallery/interactive/binaural_synthesis.ipynb

```diff
@@ -0,0 +1,327 @@
+    "    azimuth, elevation, 2\n",
+    ")\n",
+    "\n",
+    "index, _ = sources.find_nearest(desired_direction)\n",
+    "\n",
+    "sources.show(index)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the plot, the selected direction is marked with a red dot.\n",
+    "\n",
+    "With an HRIR selected, we can plot it in the time and frequency domain."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax = pf.plot.time_freq(hrirs[index], label=[\"Left ear\", \"Right ear\"])\n",
+    "ax[0].legend();\n",
+    "ax[1].legend();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating a binaural synthesis\n",
+    "\n",
+    "The binaural synthesis is done by convolving a mono, anechoic signal with an HRIR.\n",
+    "The following code shows how to do this with `pyfar`.\n",
+    "We will create a function for this to reuse it later.\n",
+    "This function gets the desired azimuth direction and selects the corresponding HRIR from the dataset.\n",
+    "Then, it convolves the input signal with the HRIR and returns the binaural signal.\n",
+    "As an additional functionality, this function also allows a gain to be applied to the binaural signal."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def convolve_with_hrir(azimuth, audio, hrirs, gain=1.0):\n",
+    "    # convert to radians\n",
+    "    azimuth = azimuth * np.pi / 180\n",
+    "\n",
+    "    # define the direction of the desired sound source\n",
+    "    desired_direction = pf.Coordinates.from_spherical_elevation(\n",
+    "        azimuth, 0, 2\n",
+    "    )\n",
+    "\n",
+    "    # find the nearest HRIRs to the desired direction\n",
+    "    index, _ = sources.find_nearest(desired_direction)\n",
+    "\n",
+    "    # apply the HRIRs to the audio signal\n",
+    "    output_audio = pf.dsp.convolve(audio, hrirs[index])\n",
+    "    output_audio = 10 ** (gain / 20) * pf.dsp.normalize(output_audio)\n",
+    "\n",
+    "    return output_audio"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use one of the audio files included in `pyfar` as an input signal.\n",
+    "\n",
+    "The following code loads the audio signal, plots the time domain and allows you to listen to it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import Audio\n",
+    "\n",
+    "castanets = pf.signals.files.castanets()\n",
+    "\n",
+    "pf.plot.time(castanets)\n",
+    "\n",
+    "Audio(castanets.time, rate=castanets.sampling_rate)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we will create a binaural synthesis from this signal."
```

Comment:
Some more explanation on what this means would be helpful (e.g., source at -90, should be appear externalized...)